ui/src/message_popup/mobile_prototype/message_components.jsx

```diff
@@ -147,7 +151,7 @@ const styles = {
     padding: 0,
     flexGrow: 0,
     flexShrink: 0,
-    flexBasis:30
+    flexBasis: 30
   },
   messageIcon: {
     width: 30,
```

Comment:
I tried changing the fixed width and height by relying on setting just the flexBasis and using margins to position it but the IconButton wave effect looks really awkward and was off-centered.